CITATION.cff

```diff
@@ -17,7 +17,7 @@ repository-code: https://github.com/bids-standard/bids-specification
 message: Please cite it as below.
 
 identifiers:
-  - description: bids-specification
+  - description: PDFs of the BIDS specification for versions 1.0.0 to 1.8.0
```

Comment:
@sappelhoff 
Can you locate the release for https://github.com/bids-standard/bids-specification-pdf-releases on zenodo: I made a release but I cannot find it

Reply:
Well I cannot see the bids matlab release either so maybe I am just too old to understand how to click buttons in a GUI.

Reply:
I think this may be on Zenodo 🤔 anyways, I deleted your tag and release and re-did them after resetting all webhooks on zenodo and github --> now it has worked.

Reply:
Nice

Reply:
by the way: minor detail but our release are vX.Y.Z, no? At least from looking at the github tags on the bids spec repo.

I think the bids spec pdf release does not have the leading "v".

Reply:
you are right. Why do we prepend a `v`? that's not really in https://semver.org/, is it?

I think I would prefer dropping the `v` prefix from now on everywhere

Reply:
I have seen this v prefix in a few places that use  "semver" but that's not systematic.
Fine with me dropping the v from now on. Most people will get it.